### ISSUE TITLE
Not working!

### ISSUE THREAD

**Issue by jeremyduanewallace** [atom/autocomplete-html#29]
Not loading in my atom text editor package is enabled but not responding

**Comment by lee-dohm** [comment]
Have you followed the suggestions in the [Debugging document](https://atom.io/docs/latest/hacking-atom-debugging)? Also, could you give the following information?

* Which version of Atom are you using?
* What OS and version are you running?
* Can you reproduce the issue after completely exiting Atom and restarting using `atom --safe`?

**Comment by orions-stars** [comment]
I seem to be having the same problem.
Issue: I've loaded Atom (then command line deleted it and reinstalled for a fresh start) but cannot see the autocomplete feature working. I have no extra packages installed. Just core. I open a new file, start typing `<htm` and nothing shows up for completion. I also tried setting up lists and a couple of other tags and nothing popped up.

My first install I went a little crazy and did what I'm not supposed to and installed emmet and another HTML auto complete package - neither of those "worked" for me as well.

Caveat: I'm a newbie. All the above is most likely something I missed.

Atom 1.8.0
Mac OSX 10.10.5
Haven't tried safe mode yet - will do.

**Comment by orions-stars** [comment]
This was solved for/by me on the atom.io discuss section. I needed to either save the file with (in my case) an `.html` extension, or let Atom know I was using HTML via the buffer. Didn't know that but between more experimenting and another Discuss member it all works for me.

See: [Autocomplete New Mac install not working](https://discuss.atom.io/t/new-mac-install-autocomplete-not-showing/30617/4?u=orions-stars)
